Assets/Scripts/CameraController.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraController : MonoBehaviour
+{
+    float rotationX = 0f;
+    float rotationY = 0f;
+
+    public GameObject Camera1;
+    public GameObject Camera2;
+
+
+    public float sensivity = 8f;
```

Comment:
start write public line after write private
```suggestion
    public GameObject Camera1;
    public GameObject Camera2;
    public float sensivity = 8f;
    float rotationX = 0f;
    float rotationY = 0f;
```